frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -311,21 +311,20 @@ export const EntitiesListBase: React.FC<InternalProps> = (
         );
     };
 
-    // InfiniteScroll will display information about loading during the request
 
+    // InfiniteScroll will display information about loading during the request
     const hasMore = props.entities.fetching || props.entities.hasMore;
+    const [sentryRef, { rootRef }] = useInfiniteScroll({
+        loading: props.entities.fetching,
+        hasNextPage: hasMore,
+        onLoadMore: getMoreEntities,
```

Comment:
There's a InfiniteScroller-specific conditional in `getMoreEntities`, which we should be able to remove now.

Reply:
Alright

Reply:
Also, sorry I'll like to ask, are you guys participating in the outreachy program this year, I plan on applying.